### PR TITLE
[Login-performance-tuning] LPS-148976

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
@dantewang, I just removed the deprecated method and made this as major version bump.

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by yuhai** [comment]
ci:test

**Comment by tinatian** [comment]
ci:test

File: portal-kernel/src/com/liferay/portal/kernel/portlet/PortletJSONUtil.java [hunk 1]
```diff
@@ -49,49 +49,79 @@ public static void populatePortletJSONObject(
...
+		_populatePortletJSONObject(
+			httpServletRequest, portletHTML, portlet, portletOnLayout,
+			jsonObject);
 	}
 
-	public static void populatePortletJSONObject(
-			HttpServletRequest httpServletRequest, String portletHTML,
-			Portlet portlet, List<Portlet> allPortlets, JSONObject jsonObject)
-		throws Exception {
+	public static void writeFooterPaths(
```
Comment (tinatian) [inline portal-kernel/src/com/liferay/portal/kernel/portlet/PortletJSONUtil.java:82]: @yuhai Why this method changed to public?

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**